### DISCUSSION TITLE
1.12.0

### DISCUSSION THREAD

**Discussion by th-ch** [pear-devs/pear-desktop#250] [General]
_Note: this release includes more than 250 commits since 1.11.0! Changelog sums up most of the changes but some might have been omitted._

- Increase top margin for MacOS
- Add custom menu support for plugins
- Last.FM plugin (by @semvis123)
- Taskbar media controls plugin (by @Araxeus)
- In-app-menu plugin (by @Araxeus)
- Improve downloader plugin (download playlist, download if player not in foreground, mutex in FFMpeg, download folder picker by @Araxeus)
- Refactored menu (by @Araxeus)
- Notification plugin: configurable notification urgency (by @SapuSeven)
- Remove 'shortcuts'+'discord' from default plugins (by @Araxeus)
- various updates of dependencies
- Bugfix in Discord plugin (by @Araxeus)
- Precise volume plugin (by @Araxeus)

<hr /><em>This discussion was created from the release <a href='https://github.com/th-ch/youtube-music/releases/tag/v1.12.0'>1.12.0</a>.</em>

**Comment by Araxeus** [comment]
Interactive Notifications for Windows were also introduced 😝

**Comment by SHU-red** [comment]
Nice Work guys!

Best YouTube Music Application

Zoom on Linux Mint 20 works!!!!
